README.md

```diff
@@ -3,43 +3,89 @@
 [![Travis branch](https://img.shields.io/travis/aragon/radspec/master.svg?style=flat-square)](https://travis-ci.org/aragon/radspec)
 [![Coveralls github branch](https://img.shields.io/coveralls/github/aragon/radspec/master.svg?style=flat-square)](https://coveralls.io/github/aragon/radspec)
 
-Radspec is a safe alternative to Ethereum's natspec[[?](#aside-why-is-natspec-unsafe)].
+Radspec is a language specification and interpreter for dynamic expressions in Ethereum's NatSpec.
```

Comment:
I think people are sometimes confused between NatSpec, the [specification](https://github.com/ethereum/wiki/wiki/Ethereum-Natural-Specification-Format), and [`natspec.js`](https://github.com/ethereum/natspec.js), its original evaluator. Can we do something the clear that up here?

Reply:
The natspec.js documentation states advertises itself as 

> Javascript Library used to evaluate natspec expressions

That seems appropriate. Radspec is also using appropriate language.

Maybe we can add very directly:

> Radspec competes with natspec.js, another library to evaluate NatSpec expressions.

Reply:
That sounds reasonable, I'll fix the remaining issues and get them merged :+1:.

Reply:
I've tried to make this as explicit as possible in https://github.com/aragon/radspec/pull/69/commits/3ac2e791d19e6cf5a2a682a7800c75e0d70dfabf.